src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -150,7 +150,24 @@ internal partial class CodeFixService : ForegroundThreadAffinitizedObject, ICode
             {
                 // sort the result to the order defined by the fixers
                 var priorityMap = _fixerPriorityMap[document.Project.Language].Value;
-                result.Sort((d1, d2) => priorityMap.ContainsKey((CodeFixProvider)d1.Provider) ? (priorityMap.ContainsKey((CodeFixProvider)d2.Provider) ? priorityMap[(CodeFixProvider)d1.Provider] - priorityMap[(CodeFixProvider)d2.Provider] : -1) : 1);
+                result.Sort((d1, d2) =>
+                {
+                    if (priorityMap.TryGetValue((CodeFixProvider)d1.Provider, out int priority1))
+                    {
+                        if (priorityMap.TryGetValue((CodeFixProvider)d2.Provider, out int priority2))
```

Comment:
It could be, but I found it much less readable. It combines too many different kinds of operations in a single statement (`out var`, ternary, and subtraction).